src/transformers/trainer_tf.py

```diff
@@ -38,7 +38,11 @@ class TFTrainer:
         args (:class:`~transformers.TFTrainingArguments`):
             The arguments to tweak training.
         train_dataset (:class:`~tf.data.Dataset`, `optional`):
-            The dataset to use for training.
+            The dataset to use for training. The dataset should yield tuples of ``(features, labels)`` where
```

Comment:
I imagine it's the same for `eval_dataset`, or the datasets passed along to `evaluate` or `predict`, so I'd copy this docstrings over there too.

Reply:
Oh yeah, I think they do expect the same format. Added.